### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by mattn**
now passgo support windows

### CODE REVIEW

**Comment by mattn** [comment]
please wait. there are some part that not closed file.

**Comment by mattn** [comment]
done

**Comment by ejcx** [comment]
Hey thanks. Very much appreciated and very good fixes. 

Any changes we can squash this to one commit, since it very repetitively points out some mistakes I made =].

File: pio/pio.go [hunk 1]
```diff
@@ -215,7 +215,7 @@ func ReadConfig() (c ConfigFile, err error) {
 // PromptPass will prompt user's for a password by terminal.
 func PromptPass(prompt string) (pass string, err error) {
 	fmt.Printf("%s:", prompt)
-	passBytes, err := terminal.ReadPassword(0)
+	passBytes, err := terminal.ReadPassword(int(os.Stdin.Fd()))
```
Comment (ejcx) [inline pio/pio.go]: excellent.

**Comment by mattn** [comment]
okay to squash. what commit message you want? :) refactoring?

**Comment by ejcx** [comment]
I prefer messages like the one below
 
```
short message describing change.

longer message containing deeper information
about what was changed. That way things are
clean and clear.
```

**Comment by ejcx** [comment]
I really appreciate the work by the way. It;s great work.

**Comment by mattn** [comment]
squashed.

**Comment by ejcx** [comment]
Sorry. I accidentally added merge conflicts. Can you resolve and then I will merge. I added functions that use the "attack" file in the `pio` package that need to be file pathed.

Everything LGTM. I really appreciate the pull.

**Comment by mattn** [comment]
rebased.

**Comment by ejcx** [comment]
Wonderful. Thanks @mattn. I appreciate the contribution.

**Pull request merged**